### PR TITLE
data: chiomavera

### PR DESCRIPTION
**Pull request by chiomavera**
## Changes proposed

-  Created my profile which entails: name, tags, bio, type, socials and links.

## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [X] My change requires changes to the documentation.
- [X] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/6243"></a>

### CODE REVIEW

File: data/chiomavera.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    "JavaScript",
+    "Tailwind",
+    "Bootstrap",
+    "CSS",
+    "HTML"
+  ],
+  "socials": [
+    { "icon": "FaTwitter", "url": "https://twitter.com/nkanmuo_vera" },
+    { "icon": "FaGithub", "url": "https://github.com/chiomavera" },
+    { "icon": "FaLinkedIn", "url": "https://www.instagram.com/pradumnasaraf" }
```
Comment (SandeepKrSuman) [inline data/chiomavera.json:17]:
```diff
+ "icon": "FaLinkedin"
```
Also, correct the URL to a valid Linkedin URL

**Review by SandeepKrSuman — CHANGES_REQUESTED** [review]
Thank you @chiomavera for your contribution 🎉
Please make the suggested changes.

File: data/chiomavera.json (hunk 1 above)
Comment (chiomavera) [reply data/chiomavera.json]: Thanks, @SandeepKrSuman for reviewing my PR.
I have made the requested changes

**Review by Sindhuinti — APPROVED** [review]

**Review by SandeepKrSuman — APPROVED** [review]
Looks Good!

**Review by khushalarora — APPROVED** [review]
Thank you for contributing :tada:

**Review by sidharthviz — APPROVED** [review]

**Review by kumarsonsoff3 — APPROVED** [review]
Looks good, Thanks for your Contribution! 🎉 

<p align="center"><a href="https://linkfree.io/docs/how-to-guides/editing"></a></p>

**Pull request merged**